src/Core/NotificationCenter/Authorization/NotificationStatusAuthorizationHandler.cs

```diff
@@ -0,0 +1,55 @@
+﻿#nullable enable
+using Bit.Core.Context;
+using Bit.Core.NotificationCenter.Entities;
+using Microsoft.AspNetCore.Authorization;
+
+namespace Bit.Core.NotificationCenter.Authorization;
+
+public class NotificationStatusAuthorizationHandler : AuthorizationHandler<NotificationStatusOperationsRequirement,
+    NotificationStatus>
+{
+    private readonly ICurrentContext _currentContext;
+
+    public NotificationStatusAuthorizationHandler(ICurrentContext currentContext)
+    {
+        _currentContext = currentContext;
+    }
+
+    protected override async Task HandleRequirementAsync(AuthorizationHandlerContext context,
```

Comment:
```suggestion
    protected override Task HandleRequirementAsync(AuthorizationHandlerContext context,
```
💭 
This is causing a build warning `Warning CS1998`. 


I think we can remove the `async` and `return Task.CompletedTask;` like they do in the example docs.

https://learn.microsoft.com/en-us/aspnet/core/security/authorization/resourcebased?view=aspnetcore-8.0#write-a-resource-based-handler